buildenv/jenkins/common/test

```diff
@@ -20,6 +20,14 @@
  * SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
  *******************************************************************************/
 
+// Set JAVA_BIN here not in the variables-functions.set_job_variables() function
+// to ensure a correct JAVA_BIN path. JAVA_BIN should be available on the 
+// machine the tests run. 
+// set_job_variables() is usually called on the master not on the node (where 
```

Comment:
Unnecessary space at end of line 24,25,26.

Reply:
removed